sdk/ai/azure-ai-projects/azure_ai_projects_tests.env

```diff
@@ -61,11 +61,11 @@ AZURE_AI_PROJECTS_TELEMETRY_TESTS_PROJECT_CONNECTION_STRING=${AZURE_AI_PROJECTS_
 ########################################################################################################################
 # Agents tests
 #
-AZURE_AI_PROJECTS_AGENTS_TESTS_PROJECT_CONNECTION_STRING=
-AZURE_AI_PROJECTS_AGENTS_TESTS_DATA_PATH=
-AZURE_AI_PROJECTS_AGENTS_TESTS_STORAGE_QUEUE=
-AZURE_AI_PROJECTS_AGENTS_TESTS_SEARCH_INDEX_NAME=
-AZURE_AI_PROJECTS_AGENTS_TESTS_SEARCH_CONNECTION_NAME=
+AZURE_AI_PROJECTS_AGENTS_TESTS_PROJECT_CONNECTION_STRING=westus.api.azureml.ms;b17253fa-f327-42d6-9686-f3e553e24763;nirovins-rg-eastus;nirovins-westus
+AZURE_AI_PROJECTS_AGENTS_TESTS_DATA_PATH=azureml://subscriptions/b17253fa-f327-42d6-9686-f3e553e24763/resourcegroups/nirovins-rg-eastus/workspaces/nirovins-westus/datastores/workspaceblobstore/paths/LocalUpload/99ccdcf221d468b60f565d5708f6ce1d/product_info_1.md
+AZURE_AI_PROJECTS_AGENTS_TESTS_STORAGE_QUEUE=https://nirovinswestus8660517702.queue.core.windows.net
```

Comment:
Do we keep your endpoint details here still?

Reply:
Fixed.